samples/suit/smp_transfer/suit/nrf54h20/root_with_binary_nordic_top_extflash.yaml.jinja2

```diff
@@ -239,20 +249,30 @@ SUIT_Envelope_Tagged:
       - suit-send-sysinfo-success
       - suit-send-sysinfo-failure
 {%- endif %}
-{%- if radio is defined %}
+{%- if application is defined %}
     - suit-directive-override-parameters:
-        suit-parameter-uri: '#{{ radio['name'] }}'
+        suit-parameter-uri: '#{{ application['name'] }}'
         suit-parameter-image-digest:
           suit-digest-algorithm-id: cose-alg-sha-256
           suit-digest-bytes:
-            envelope: {{ artifacts_folder ~ radio['name'] }}.suit
+            envelope: {{ artifacts_folder ~ application['name'] }}.suit
     - suit-directive-fetch:
       - suit-send-record-failure
-    - suit-condition-image-match:
-      - suit-send-record-success
-      - suit-send-record-failure
-      - suit-send-sysinfo-success
-      - suit-send-sysinfo-failure
+    # In the case of streaming operations it is worth to retry them at least once.
```

Comment:
As discussed, this change seems a bit pointless for the root manifest as there actually is no streaming - just acquiring the digest fom a dependency manifest, which has to be in the MRAM memory anyway.
It was changed in the normal root manifest, but  I will change it back in an upcoming PR.

Reply:
Removed!